dockers/docker-orchagent/orchagent.sh

```diff
@@ -30,6 +30,8 @@ elif [ "$HWSKU" == "INGRASYS-S8900-54XC" ]; then
     ORCHAGENT_ARGS+="-m $MAC_ADDRESS"
 elif [ "$HWSKU" == "INGRASYS-S8900-64XC" ]; then
     ORCHAGENT_ARGS+="-m $MAC_ADDRESS"
+elif [ "$HWSKU" == "Accton-AS7712-32X" ]; then
```

Comment:
Yes. The name of the `<HARDWARE_SKU>` directory should match your `$HWSKU` exactly, which, in this case is `Accton-AS7712-32X`.